### ISSUE TITLE
Unit tests failing on branch main: CategoryClientTest.getSubCategoryListFound etc

### ISSUE THREAD

**Issue by nicolas-raoul** [commons-app/apps-android-commons#5921]
commit f011abef1d480a2393818be60dd142309bdb5f93
```
CategoryClientTest > getSubCategoryListFound FAILED
    java.lang.AssertionError at CategoryClientTest.kt:193
        Caused by: java.lang.NullPointerException at CategoryClientTest.kt:192

CategoryClientTest > searchCategoriesFound FAILED
    java.lang.AssertionError at CategoryClientTest.kt:40
        Caused by: java.lang.NullPointerException at CategoryClientTest.kt:39

CategoryClientTest > getParentCategoryListFound FAILED
    java.lang.AssertionError at CategoryClientTest.kt:171
        Caused by: java.lang.NullPointerException at CategoryClientTest.kt:170

CategoryClientTest > searchCategoriesForPrefixFound FAILED
    java.lang.AssertionError at CategoryClientTest.kt:70
        Caused by: java.lang.NullPointerException at CategoryClientTest.kt:69

CategoryClientTest > getCategoriesByNameFound FAILED
    java.lang.AssertionError at CategoryClientTest.kt:106
        Caused by: java.lang.NullPointerException at CategoryClientTest.kt:105
```

**Comment by rohit9625** [comment]
Surprisingly only a single change fixed all categories related errors.

**Comment by rohit9625** [comment]
I have fixed several more tests. Now, only 6 tests from the `ImageProcessingServiceTest` file are failing. Should I add the changes to the same branch for this issue?

**Comment by nicolas-raoul** [comment]
Same branch is fine.
Thanks a lot! 🙂

**Comment by rohit9625** [comment]
Please close this issue as fixed via #5947
